en-docs: Improve Install Instruction Formating (#17996)
kubernetes/website: content/en/docs/tasks/tools/install-kubectl.md

### Install using other package management

{{< tabs name="other_kubectl_install" >}}
{{% tab name="Snap" %}}

> If you are on Ubuntu or another Linux distribution that support [snap](https://snapcraft.io/docs/core/install) package manager, kubectl is available as a [snap](https://snapcraft.io/) application.

Reviewer: It's fine to retain the existing text. I think this might work better:
```suggestion
If you use a Linux distribution that supports [snap](https://snapcraft.io/docs/core/install) package management, such as Ubuntu, you can install kubectl using `snap`:
```

Author: Reverted back to the original text.

> ```

Reviewer: Mark this as POSIX shell code (append `shell` to the three backticks )

Author: Added.

> sudo snap install kubectl --classic

Reviewer: Aside: we probably don't need `sudo` here.

Author: Removed

kubectl version
```
{{% /tab %}}
{{% tab name="Homebrew" %}}

> If you are on Linux and using [Homebrew](https://docs.brew.sh/Homebrew-on-Linux) package manager, kubectl is available for [installation](https://docs.brew.sh/Homebrew-on-Linux#install).

Reviewer: It's fine to retain the existing text. I think this might work better:
```suggestion
If you are running Linux and using the [Homebrew](https://docs.brew.sh/Homebrew-on-Linux) package manager, you can [install kubectl using Homebrew](https://docs.brew.sh/Homebrew-on-Linux#install):
```

Author: Reverted back to the original text.

> ```

Reviewer: Mark this as POSIX shell code (append `shell` to the three backticks )

Author: Added.
